src/content/docs/en/guides/assets.mdx

```diff
@@ -0,0 +1,277 @@
+---
+title: Assets (Experimental)
+description: Learn how to enable experimental asset support in Astro.
+i18nReady: false
+---
+
+**Built-in Asset Support** is enabled in Astro behind an experimental flag. This built-in feature will eventually replace the optional `@astrojs/image` integration.
+
+The new assets experience currently features:
+
+- A new built-in `<Image />` component
+- Relative images with automatic optimization in Markdown and MDX files
+- Integration with content collections
+- Improved error messages and types
+
+:::caution
+Assets are an experimental Astro feature. This API is subject to change before it is marked as stable.
+:::
+
+## Enabling Assets in your Project
+
+Enabling assets may cause some breaking changes in your project. It may also require some manual changes to take advantage of new features.
+
+Please check every section below to avoid errors and to get the most out of using the experimental assets option!
+
+### Update Config
+
+To enable built-in asset support, add the following lines to your `astro.config.mjs` configuration file:
+
+```js title="astro.config.mjs" ins={4-6}
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+ experimental: {
+  assets: true
+ }
+});
+```
+
+### Move your images to `src/assets/`
+
+Create an `src/assets/` folder, which Astro will recognize as your new assets folder.
+
+We **strongly recommend** storing all images to be optimized in the `src/assets/` directory. This location is optional for now, but may become the required location for all asset types in the future.
```

Comment:
Maybe there should be a bit more nuance in this section, as to not scare users who want to put their images for Markdown in `content`. 

It's ultimately unlikely that we'll force assets to be in the `src/assets` folder, it should be seen more as "the approved folder" than "the folder you need to use"

Reply:
I can back down on this! I was also hoping that, because it's the approved path, if we got people using it during the experimental phase, that would be easier for troubleshooting/dog fooding etc.